docs/pages/schema_props.md

```diff
@@ -0,0 +1,39 @@
+| schema:schemaVersion     | Indicates (by URL or string) a particular version of a schema used in some CreativeWork. For example, a document could declare a schemaVersion using an URL such as http://schema.org/version/2.0/ if precise indication of schema version was required by some application. | reproschema:Activity | ['schema:Text', 'schema:URL']          |
+| schema:version           | The version of the CreativeWork embodied by a specified resource.                                                                                                                                                                                                            | reproschema:Activity | ['schema:Text', 'schema:Number']       |
+| schema:citation          | A citation or reference to another creative work, such as another publication, web page, scholarly article, etc.                                                                                                                                                             | reproschema:Activity | ['schema:Text', 'schema:CreativeWork'] |
+| reproterms:preamble      | The preamble for the instrument.                                                                                                                                                                                                                                             | reproschema:Activity | schema:Text                            |
+| reproterms:order         | An array to describe the order in which the Fields of an instrument appear in the ui. It represents a list of node ids of the items and not the variable names.                                                                                                              | reproschema:Activity | schema:URL                             |
+| reproterms:shuffle       | An element (bool) to determine if the list of items is shuffled or in order.                                                                                                                                                                                                 | reproschema:Activity | schema:boolean                         |
+| reproterms:addProperties | An array of objects to describe the various properties added to assessments and fields.                                                                                                                                                                                      | reproschema:Activity | reproschema:AddProperties              |
+| reproterms:scoringLogic  | An array of objects indicating score in an activity or protocol and the score Field item                                                                                                                                                                                     | reproschema:Activity | reproschema:ScoringLogic               |
+# Properties of Field schema
```

Comment:
Am I missing something or shouldn't the next section give details about  the Item schema ?

To me it looks like this is a duplicate of the schema of activity.

Reply:
oops! Sorry I will update that.